<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="addWarship">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Add warships</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addYacht">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Add yachts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSubmarine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>3. Add submarines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handleGame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4. Start game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handleExit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>5. Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
